package.json

```diff
@@ -11,6 +11,7 @@
     "production": "rimraf dist && webpack --config ./webpack.prod.js --progress --colors"
   },
   "devDependencies": {
+    "@types/jquery": "^3.3.28",
```

Comment:
Oof, I'd really rather we not add this. jQuery isn't necessary. Looking at the code, it's only used for `$.ajax`, which is easily replaced by `window.fetch` or a wrapper like wretch.

Reply:
That's fine, I just find jquery ajax calls super simplistic, but I can refactor to use `window.fetch`